.github/workflows/main.yml

```diff
@@ -0,0 +1,51 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  api:
+    name: Integrate api
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./api
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-api-node-modules
+        with:
+          path: '**/node_modules'
```

Comment:
Will this cache the `node_modules/` under `www/` too? Don't we just want `./api/node_modules`?

Reply:
Because the working directory is in `./api/node_modules` this will only target the current `node_modules`

Reply:
I might be mistaken, but I think the working directory is only set to `api` for `run` jobs not `use` jobs. Not a huge deal either way.

Reply:
See this https://stackoverflow.com/a/63122434/10142748

Reply:
Ah good catch, I was wondering why it didn' trigger a cache miss in some cases, let me try this out. Thanks for pointing out the resource!